Vagrantfile

```diff
@@ -1,6 +1,18 @@
 require 'yaml'
 require 'ipaddr'
 
+# Proc settings merger
+settings_merger = proc {
+    |key, v_default, v_env|
```

Comment:
Does it work for netested levels? So far we've only two levels..

Reply:
yes it is recursive